resource/js/navbar-ui-languages.js

```diff
@@ -0,0 +1,19 @@
+function setLangCookie (lang) {
+  // The cookie path should be relative if the baseHref is known
+  let cookiePath = '/'
+  if (window.SKOSMOS.baseHref && window.SKOSMOS.baseHref.replace(window.origin, '')) {
+    cookiePath = window.SKOSMOS.baseHref.replace(window.origin, '')
+  }
+  document.cookie = `SKOSMOS_LANGUAGE=${lang};path=${cookiePath}`
```

Comment:
This only sets the cookie for the duration of the session. I think the intent was to remember the language setting for the next time, so keeping it only for this session isn't enough. Skosmos 2 sets the cookie for 365 days, the same should be done here. Also, Skosmos 2 uses the "SameSite=Lax" parameter, is that needed here as well?